docs/_series/kidphocks.md

```diff
@@ -35,6 +35,29 @@ subs:
     image: https://i.imgur.com/MaVDI4U.gif
     supply: 99
   

```

Comment:
You're missing "-" at the beginning of each block.

Reply:
ok corrected, thx